### PR TITLE
Fix newly created user is not added into global-viewer group

### PR DESCRIPTION
**Pull request by Rayhanga**
Remove validation for `role_id` attribute at GroupUser model.

Fixes https://github.com/BaritoLog/BaritoMarket/issues/246

### CODE REVIEW

File: app/models/group_user.rb [hunk 1]
```diff
@@ -3,6 +3,6 @@ class GroupUser < ActiveRecord::Base
   belongs_to :user
   belongs_to :role, class_name: 'AppGroupRole'
 
-  validates :group_id, :user_id, :role_id, presence: true
```
Comment (fadlinurhasan) [inline app/models/group_user.rb:6]: if we remove role validation here, is there any default role for each user when added to groupUser?

**Review by fadlinurhasan — COMMENTED** [review]
Add a comment

File: app/models/group_user.rb (hunk 1 above)
Comment (Rayhanga) [reply app/models/group_user.rb:6]: > if we remove role validation here, is there any default role for each user when added to groupUser?

Based on below codes at `app/controllers/app_group_users_controller.rb `
https://github.com/BaritoLog/BaritoMarket/blob/1b304434783cbc9bd272305d9005a0df8985a1ca/app/controllers/app_group_users_controller.rb#L1-L19
Both in `set_role` and overridden `create` method has `member` as their default role.

Even though it's hard-coded, based on the `db/seeds.rb`.

https://github.com/BaritoLog/BaritoMarket/blob/1b304434783cbc9bd272305d9005a0df8985a1ca/db/seeds.rb#L4-L6

Since it's inside `db/seeds.rb` there should always be an AppGroupRole called `member`.

TL;DR: I think yes, there should be a default role called `member` for each user when added to groupUser (although I'm unsure if `db/seeds.rb` will always initialize said role or not, but based on my testing result on my local machine it should work as expected).

Comment (fadlinurhasan) [reply app/models/group_user.rb:6]: It is controller that will use when user add user to group by UI.
seed only called once when you run it like migrate process.

You need to keep the validation and add role when new user added to group or remove validation role but need to override the value under models.

Comment (Rayhanga) [reply app/models/group_user.rb:6]: Ah ok I see, will try to resolve this tomorrow, I'll try explore some of the suggestions on my own for now, thank you!

**Review by fadlinurhasan — APPROVED** [review]
